include/zenoh/api/hello.hxx

```diff
@@ -50,7 +50,7 @@ class Hello : public Owned<::z_owned_hello_t> {
         ::z_hello_locators(interop::as_loaned_c_ptr(*this), &out);
         auto out_loaned = ::z_loan(out);
 #else
-        auto out_loaned = ::z_hello_locators(interop::as_loaned_c_ptr(*this));
+        auto out_loaned = ::zp_hello_locators(interop::as_loaned_c_ptr(*this));
```

Comment:
if z_hello_locators are available in both pico and c, it is probably better to remove conditionals and just use common version for both zenoh-c and pico cases.

Reply:
It will decrease performance of zenoh-pico version